app/gui/controller/double-representation/src/module.rs

```diff
@@ -282,6 +282,28 @@ impl QualifiedName {
             self.id.segments.pop();
         }
     }
+
+    /// Check if the name refers to some Library's top module.
```

Comment:
Why library is written with upper-case? It is not a name in this context, is it?